components/omega/doc/userGuide/Tracers.md

```diff
@@ -0,0 +1,61 @@
+(omega-user-tracers)=
+
+# Tracers
+
+Tracers refer to either heat (temperature) or material carried by a fluid
+parcel (e.g., salt, chemical, or biological constituents).
+
+## Updating tracer defintions in `TracerDefs.inc` file
+
+To manage tracer definitions, users will update the `TracerDefs.inc` file
+located in the `omega/src/ocn` directory of E3SM. The file contains
+Tracer index variables and `defineAllTracers` C++ function defintion that
+contains the calls to `define` function per each tracer as shown below:
+
+```c++
+inline static I4 IndxTemp             = Tracers::IndxInvalid;
+inline static I4 IndxSalt             = Tracers::IndxInvalid;
+inline static I4 IndxMyBGCTracer      = Tracers::IndxInvalid;
+
+// Tracer definitions packaged in a defineAllTracers function
+static void defineAllTracers() {
+
+   define("Temp",                            ///< [in] Name of tracer
+          "Potential Temperature",           ///< [in] Long name or description
```

Comment:
noting here and in similar places, for TEOS-10 (new EOS) this should actually be `Conservative Temperature` not `Potential Temperature` and `sea_water_conservative_temperature` for the CF name

Similarkly salinity is now `Absolute Salinity` and not `Practical Salinity`  units are g/kg instead of psu

Reply:
This could be changed later.